developer/src/kmc-model-info/src/model-info-compiler.ts

```diff
@@ -1,163 +1,209 @@
 /**
- * Merges a source .model_info file with metadata extracted from .kps file and
+ * Builds a source .model_info file with metadata extracted from .kps file and
  * compiled files to produce a comprehensive .model_info file.
  */
 
 import { minKeymanVersion } from "./min-keyman-version.js";
 import { ModelInfoFile } from "./model-info-file.js";
 import { CompilerCallbacks, KmpJsonFile } from "@keymanapp/common-types";
 import { ModelInfoCompilerMessages } from "./messages.js";
+import { validateMITLicense } from "@keymanapp/developer-utils";
+
+const HelpRoot = 'https://help.keyman.com/model/';
 
 /* c8 ignore start */
-export class ModelInfoOptions {
+export class ModelInfoSources {
   /** The identifier for the model */
   model_id: string;
 
   /** The data from the .kps file, transformed to kmp.json */
   kmpJsonData: KmpJsonFile.KmpJsonFile;
 
   /** The path in the keymanapp/lexical-models repo where this model may be found (optional) */
```

Comment:
```suggestion
  /** The path in the keymanapp/lexical-models repo where this model may be found */
```

sourcePath looks required now?